server/views/pages/service-updates.njk

```diff
@@ -2,26 +2,25 @@
 {% set pageName = "Service updates" %}
 
 {% block content %}
-  {% if whatsNewVersion >= 20241120 %}
```

Comment:
Are we still showing only the banners after the version in the config in here?

Reply:
Yes now it filters the list in the serviceUpdateController.ts to only versions less than or equal to the version in the config